Formula/kde-kdoctools.rb

```diff
@@ -1,8 +1,9 @@
 class KdeKdoctools < Formula
   desc "Create documentation from DocBook"
   homepage "https://api.kde.org/frameworks/kdoctools/html/index.html"
-  url "https://download.kde.org/stable/frameworks/5.76/kdoctools-5.76.0.tar.xz"
-  sha256 "84ea7974d741e6261e8c269750367a00375c6111dbc542e917647d0267337ae4"
+  url "https://download.kde.org/stable/frameworks/5.77/kdoctools-5.77.0.tar.xz"
+  sha256 "a78c44534f85824ed8168670d2880d3735083129635109dfe54fc21f16fb6d44"
+  license all_of: ["BSD-3-Clause", "GPL-2.0-or-later", "LGPL-2.1-only", "LGPL-2.1-or-later", "LGPL-3.0-only"]
```

Comment:
[`src/checkXML5.cpp`](https://invent.kde.org/frameworks/kdoctools/-/blob/master/src/checkXML5.cpp) and [`src/docbookl10nhelper.cpp`](https://invent.kde.org/frameworks/kdoctools/-/blob/master/src/docbookl10nhelper.cpp) have a line saying:

```
SPDX-License-Identifier: LGPL-2.1-only OR LGPL-3.0-only OR LicenseRef-KDE-Accepted-LGPL
```

[`LicenseRef-KDE-Accepted-LGPL`](https://invent.kde.org/frameworks/kdoctools/-/blob/master/LICENSES/LicenseRef-KDE-Accepted-LGPL.txt) seems to be defined as `LGPL` version 3.0 or later versions that are approved. Since versions need to be approved, it doesn't seem like `LGPL-3.0-or-later` is really appropriate.

These are the only references to these three licenses, so that makes me think we should use:

```ruby
  license all_of: [
    "BSD-3-Clause",
    "GPL-2.0-or-later",
    "LGPL-2.1-or-later",
    any_of: ["LGPL-2.1-only", "LGPL-3.0-only"],
  ]
```